api/src/main/java/com/community/api/endpoint/avisoft/controller/ServiceProvider/ServiceProviderRankController.java

```diff
@@ -43,5 +43,17 @@ public ResponseEntity<?> getAllServiceProviderRank() {
         return responseService.generateResponse(HttpStatus.OK, "Service Provider Test Rank List Retrieved Successfully", serviceProviderTestRankList);
     }
 
+    @GetMapping("/get-score-card/{serviceProviderId}")
+    public ResponseEntity<?> getScoreCardToServiceProvider(@PathVariable Long serviceProviderId) {
+        try {
+            Map<String,Integer> scoreCard =serviceProviderRankService.getScoreCard(serviceProviderId);
```

Comment:
you created this action
now we can add MethodArgumentTypeMismatchException in catch

 } catch (IllegalArgumentException e) {
        return ResponseService.generateErrorResponse(e.getMessage(), HttpStatus.BAD_REQUEST);
    }